config.py.dist

```diff
@@ -1,32 +1,35 @@
 DB_KEY = 'abc:123'
 
-CON_STRING = ""
+CON_STRING = ''
 
 
-DEVELOPMENT = ""
-CODE_SQL = ""
+DEVELOPMENT = ''
+CODE_SQL = ''
 
-ALL_SQL = ""
+ALL_SQL = ''
 
-TABLE_NAME = ""
+TABLE_NAME = ''
 
-KEY_COLUMN = ""
+KEY_COLUMN = ''
 
-CAPS_GS_SEM_RECIPIENTS = ""
-BCC = ""
+CAPS_GS_SEM_RECIPIENTS = ''
+BCC = ''
 
-WSDL = ""
+WSDL = ''
 AUTH = {'username': '', 'password': ''}
 SITE_ID = ''
 STAGING_DESTINATION_ID = ''
 
 PUBLISHSET_ID = ''
 
-secret_key = ""
-SECRET_KEY = ""
+secret_key = ''
+SECRET_KEY = ''
 
 XML_URL = ''
 
 RECIPIENTS = ['']
 
 RAVEN_URL = ''
+
+# Has to be absolute path, not relative
+BANNER_HASHES_CSV_PATH = ''
```

Comment:
Might be worth changing the name of this or adding a comment to specify what this is used for?

Reply:
If it were to be renamed, it would be a crazy long name. I'll opt to add a comment in .dist to explain what it is/where it gets used.

Reply:
`BANNER_HASHES_AUDIT_CSV_PATH`?